lcls_tools/superconducting/scLinac.py

```diff
@@ -18,24 +18,26 @@ def __init__(self, cavity):
         # type: (Cavity) -> None
         self.cavity: Cavity = cavity
         self._pv_prefix = self.cavity.pv_addr("SSA:")
+        self.shared_pvs = ["PSVoltSetpt1", "PSVoltSetpt2", "StatusMsg",
+                           "PowerOn", "PowerOff", "FaultReset", "NRP_PRMT"]
```

Comment:
Does this also need to account for the pv associated with our three letter code SSA? "ACCL:L1B:H110:SSA:FaultSummary.SEVR"?